### PR TITLE
Merge work-branch with master

### PR DESCRIPTION
**Pull request by simonsvale**
merge

### CODE REVIEW

File: Client/client_final.py [hunk 1]
```diff
@@ -273,6 +288,17 @@ def video(self):
         print("Starting Video Process")
         self.process = subprocess.Popen(self.ffmpeg_cmd, stdout=subprocess.PIPE)
 
+    def status_information(self):
```
Comment (Mikkel-Coder) [inline Client/client_final.py:291]: This is rather odd.

File: Client/client_final.py [hunk 2]
```diff
@@ -1,14 +1,12 @@
-import subprocess, requests, socket, threading, PySimpleGUI as sg
+import subprocess, requests, socket, threading, PySimpleGUI as sg, copy
 from pynput import keyboard
 from time import sleep
 
 
-BACKEND_URL = 'http://89.150.129.29:8000/v1/api/frontend'
-BACKEND_IP = '89.150.129.29'
+BACKEND_URL = 'http://00.00.00.00:8000/v1/api/frontend'
```
Comment (Mikkel-Coder) [inline Client/client_final.py:6]: What is this not an f-string?

**Review by Mikkel-Coder — APPROVED** [review]
Why are we doing this?

File: Client/client_final.py (hunk 2 above)
Comment (Frier03) [reply Client/client_final.py:6]: No comments

File: Client/client_final.py (hunk 1 above)
Comment (Mikkel-Coder) [reply Client/client_final.py:291]: @Frier03 Please comment on this.

Comment (Frier03) [reply Client/client_final.py:291]: Its maybe a function to debug. Me dont know. Sus code

**Pull request merged**